README.md

```diff
@@ -2,7 +2,7 @@
 
 # BookStore
 
-> I built a webpage where you can add books.
+> I built a webpage using React & Redux where you can add books and matches with [Zeplin project](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378)
 
 ### Live GitHub Page
 Not available
```

Comment:
- [ ] [optional] It would have been nice if you added how to clone and run the project locally in the getting started section.

Reply:
Thank you I just implemented what you suggested:100: